tests/integration/strategy_tests.cpp

```diff
@@ -325,7 +338,7 @@ TEST(SizeLimiter, Host)
 {
   auto& rm = umpire::ResourceManager::getInstance();
 
-  auto alloc = rm.makeAllocator<umpire::strategy::SizeLimiter>(
+  auto alloc = rm.makeAllocator<umpire::strategy::SizeLimiter, false>(
```

Comment:
Could you back this change out?

Reply:
This fixes the test on ray, but I'll remove if you want.
https://github.com/LLNL/Umpire/pull/84#discussion_r268299826

Reply:
Yup, I'll dig into it.

Reply:
Should I remove it in the meantime?

Reply:
Yes please.

Reply:
Done.

Reply:
FYI it passes in a release build, but still fails in debug.